Added Supabase Streaming Filter Limitation (#209)
FlutterFlow/flutterflow-documentation: docs/resources/control-flow/backend-logic/backend-query/query-collection.md

    </iframe>
</div>

:::warning[Limitations of Supabase Streaming with Filters]
When using Supabase query with real-time updates enabled, you have the following limitations:

> 1. **Only One Filter is Supported:** Supabase streaming supports only a single filter. Combining multiple filters (e.g., `isActive = true AND city = 'Los Angeles'`) is not allowed.

Reviewer: Should be a bullet list instead of ordered since this is not a ordered list right?

Author: Yes, updated.

> 2. **Delete Events Are Not Filterable:** Streaming queries do not detect deletions, even if the deleted row matches the filter condition. For example, If you are streaming rows with the filter `city = 'New York’` and a row is deleted, the query output will not reflect the deletion.

Reviewer: ```suggestion
2. **Delete Events are not Filterable:** Streaming queries do not detect deletions, even if the deleted row matches the filter condition. For example, If you are streaming rows with the filter `city = 'New York’` and a row is deleted, the query output will not reflect the deletion.
```

> 3. **Updates That Remove Rows from Filteres Are Not Tracked:** Changes that make a row no longer match the filter condition (e.g., updating `isActive` from `true` to `false`) will not trigger an update in the query output.

Reviewer: ```suggestion
3. **Updates that remove Rows from Filters are not Tracked:** Changes that make a row no longer match the filter condition (e.g., updating `isActive` from `true` to `false`) will not trigger an update in the query output.
```

> For more details, refer the limitations [**here**](https://supabase.com/docs/guides/realtime/postgres-changes?queryGroups=language&language=js&queryGroups=database-method&database-method=dashboard#delete-events-are-not-filterable).

Reviewer: ```suggestion
For more details, refer to the limitations mentioned in the [**official Supabase docs**](https://supabase.com/docs/guides/realtime/postgres-changes?queryGroups=language&language=js&queryGroups=database-method&database-method=dashboard#delete-events-are-not-filterable).
```
